bindings/python/pydeck/tests/browser/test_html_renderer.py

```diff
@@ -3,7 +3,6 @@
 from pydeck import Deck, Layer, ViewState
 
 from ..const import FIXTURE_STRING
-from .screenshot_utils import go_to_page_and_screenshot
```

Comment:
Should this file be dropped?

Reply:
Using it for local development tests at the moment